docs/visualization/visualize_neuropixel_probes.ipynb

```diff
@@ -74,23 +74,27 @@
      "name": "stderr",
```

Comment:
<p>typo. can make the comment just the stuff which inside the parenthesis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>